spec/models/recruitment_cycle_spec.rb

```diff
@@ -1,6 +1,23 @@
 require 'rails_helper'
 
 RSpec.describe RecruitmentCycle do
+  describe '.cycle_string' do
+    it 'throws an error when a cycle does not exist for the specified year' do
```

Comment:
@cwrw added a spec as well to make sure we maintain this functionality in case of any future updates.